src/styles/_variables.scss

```
@@ -15,11 +15,10 @@ $thumbnail_background_color_alt: #33343e;
 $preview_description_font_color: #90909c;
 $buttons_background: #1d1e27;
 
-//Custom styles
 $custom_links_color: #67f7b8;
 $search_icon_hover_color: $white_color;
 $search_icon_backgound: #52528b;
-$article_description_color: #96969b;
 $comment_text_color: #8f9095;
 $tip_note_color: #babac0;
 $text_content_color: #d2d2d4;
+$warning_font_color: #856404;
```

```suggestion
$warning_font_color: #ffdc50;
```
We need to use only design consistent colors. This one is approved for warning tough it is not yet implemented elsewhere. It is for "site badges" - warning, we can use it sitewise

---

Thanks for providing it.